source/git-util.js

```diff
@@ -77,6 +85,15 @@ exports.verifyCurrentBranchIsReleaseBranch = async releaseBranch => {
 	}
 };
 
+exports.isHeadDetached = async () => {
+	try {
+		await execa('git', ['symbolic-ref', '-q', 'HEAD']);
```

Comment:
Use long-flag.

Reply:
Ok, but actually it works the same without the flag `-q` (quiet), so I just removed it.